package.json

```diff
@@ -26,28 +26,28 @@
     "yalc-check": "./dist/bin/yalc-check.js"
   },
   "dependencies": {
-    "@babel/cli": "^7.17.6",
-    "@babel/core": "^7.17.5",
-    "@babel/preset-env": "^7.16.11",
-    "@babel/preset-typescript": "^7.16.7",
-    "@commitlint/cli": "^16.2.1",
-    "@commitlint/config-lerna-scopes": "^16.2.2",
+    "@babel/cli": "^7.17.10",
+    "@babel/core": "^7.18.0",
+    "@babel/preset-env": "^7.18.0",
+    "@babel/preset-typescript": "^7.17.12",
+    "@commitlint/cli": "^17.0.0",
+    "@commitlint/config-lerna-scopes": "^17.0.0",
     "@jupiterone/eslint-config": "^2.0.0",
     "@ovos-media/ts-transform-paths": "^1.7.18-1",
     "@philidem/rewrite-relative-paths": "^1.1.2",
-    "@types/jest": "^27.4.1",
+    "@types/jest": "^27.5.1",
     "babel-jest": "^27.5.1",
     "babel-plugin-module-resolver": "^4.1.0",
-    "eslint": "^8.11.0",
-    "husky": "^7.0.4",
+    "eslint": "^8.16.0",
+    "husky": "^8.0.1",
```

Comment:
For others wondering what may have changed: https://github.com/typicode/husky/releases/tag/v8.0.0